.github/workflows/gpg_signature.yml

```diff
@@ -0,0 +1,19 @@
+name: GPG signature
+on: [push]
+jobs:
+  build:
+    name: GPG signature
+    strategy:
+      matrix:
```

Comment:
I think there is no value running this in matrix. One OS should be enough and I would use some smaller image than ubuntu...

Reply:
ubuntu is what's available on GH Actions, and the matrix is just one job, but yeah it could be simplified to not use a matrix for clarity.

Reply:
The matrix allow us to add ruby versions to be included in tests. For a while, only ruby 2.7 is defined, but we can add later 3.0, 3.1, 2.6 and so on 🍻